examples/maps/README.md

```diff
@@ -0,0 +1,40 @@
+# MapReader and maps
+
+- [Tutorials](#tutorials)
+  - [classification_one_inch_maps_001](https://github.com/Living-with-machines/MapReader/tree/main/examples/maps/classification_one_inch_maps_001)
```

Comment:
Can there be a landing page for this Tutorial that gives me some space to walk people through the task of analyzing 1-inch maps, how to work through the different notebooks in order, and what it might look like to change this to a new task?

Reply:
Sure, just add a README file here: https://github.com/Living-with-machines/MapReader/tree/restructure-readme-kh/examples/maps/classification_one_inch_maps_001

Reply:
Great, will do this.